mobile/src/navigations/onboarding-navigation/components/step/step.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+
+import { Pressable, Text, View } from '~/bundles/common/components/components';
+import { Color, IconName, TextCategory } from '~/bundles/common/enums/enums';
+import { useMemo } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type StepState = {
+    FOCUSED: boolean;
+    COMPLETED: boolean;
+    DISABLED: boolean;
+};
```

Comment:
Maybe I can name it as StepState because we already have enum which is called AppEnvironment (in shared folder and we have multiple exports of the same name)